app/assets/stylesheets/application.css.scss

```diff
@@ -91,6 +91,28 @@ input {
   padding-top: 12%;
 }
 
+.jumbotron > #form-c {
+  background: none !important;
+  text-align: center;
+}
+
+.jumbotron > form > input[type="text"] {
+ border-radius: 4px;
+ border: 0px;
+ width: 30%;
+ margin-top: 1px;
+ height: 30px;
+ text-align: center;
+}
+
+#form-ul > form > input[type="text"] {
```

Comment:
Avoid using id selectors<br>Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3